web/src/main/java/org/cbioportal/web/util/ClinicalDataIntervalFilterApplier.java

```diff
@@ -22,9 +21,8 @@
     @Autowired
     public ClinicalDataIntervalFilterApplier(PatientService patientService,
                                              ClinicalDataService clinicalDataService,
-                                             SampleService sampleService,
                                              StudyViewFilterUtil studyViewFilterUtil) {
-        super(patientService, clinicalDataService, sampleService, studyViewFilterUtil);
+        super(patientService, clinicalDataService, studyViewFilterUtil);
```

Comment:
kinda related to my previous comment for not join sample table. I think we should keep the sampleService and query samplesService to filter patients by samples.